guides/common/modules/proc_loading-the-default-openscap-contents.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Loading_the_Default_OpenSCAP_Contents_{context}"]
+= Loading the Default OpenSCAP Contents
+
+Use this procedure to load the default OpenSCAP contents nn {ProjectServer}.
+This procedure ensures that data streams available in `/usr/share/xml/scap/ssg/content/` are loaded and assigned to all organizations and locations.
+
+.Prerequisites
+* Your user account has the `create_scap_contents` permission.
+* Ensure that the *SCAP Security Guide* package is installed on {ProjectServer}:
++
+ifdef::foreman-deb[]
```

Comment:
Interesting we have this because we don't package `foreman_openscap` on Debian.

Reply:
These packages come with the OS, not Foreman.

Reply:
Ohhh, you mean that Openscap plugin is not supported on Debian?

Reply:
Exactly. As discussed on IRC: you can hide the entire guide for Debian or have some placeholder that says "not supported".